.github/workflows/build.yml

```diff
@@ -249,10 +249,45 @@ jobs:
           set PATH=%PATH%;%GITHUB_WORKSPACE%\ACE_TAO\ACE\lib;%GITHUB_WORKSPACE%\OpenDDS\lib;%GITHUB_WORKSPACE%\tools\v8stubs"
           echo "PATH = %PATH%"
           msbuild -p:Configuration=Debug,Platform=x64 -m node_opendds.sln
+      - name: 'Test (windows)'
+        if: ${{ matrix.m.os == 'windows-2019' }}
+        shell: cmd
+        run: |-
+          cd OpenDDS
+          call setenv.cmd
+          cd ..
+          set /p V8_ROOT=<v8_path.txt
+          set V8_ROOT=%V8_ROOT%\..\..
+          echo "V8_ROOT = %V8_ROOT%"
+          set PATH=%PATH%;%GITHUB_WORKSPACE%\ACE_TAO\ACE\lib;%GITHUB_WORKSPACE%\OpenDDS\lib;%GITHUB_WORKSPACE%\tools\v8stubs"
+          echo "PATH = %PATH%"
           cd test
-          perl run_test.pl cpp2node
+          REM perl run_test.pl cpp2node # re-enable when fixed
+          REM if %errorlevel% neq 0 exit /b %errorlevel% # re-enable when fixed
           REM perl run_test.pl node2cpp # re-enable when fixed
+          REM if %errorlevel% neq 0 exit /b %errorlevel% # re-enable when fixed
           REM perl run_test.pl node2node # re-enable when fixed
+          REM if %errorlevel% neq 0 exit /b %errorlevel% # re-enable when fixed
           perl run_test.pl cpp2node --rtps
-          REM perl run_test.pl node2cpp --rtps # re-enable when fixed
-          REM perl run_test.pl node2node --rtps # re-enable when fixed
+          if %errorlevel% neq 0 exit /b %errorlevel%
+          perl run_test.pl node2cpp --rtps
+          if %errorlevel% neq 0 exit /b %errorlevel%
+          perl run_test.pl node2node --rtps
+      - name: 'Test Secure (windows)'
```

Comment:
Could this follow the same pattern as Linux (or the other way around) so there's one approach to configurable security-enabled testing?